modules/installation/pages/client-requirements.adoc

```diff
@@ -36,65 +36,132 @@ For non-{suse} operating systems, including {rhel}, {centos}, and {oracle}, only
 | {check}
 
 | {sle} 11
-| {x86}, {x86_64}, {ppc64}, {ibmz}
+| {x86}, {x86_64}, {ipf}, {ppc64}, {ibmz}
 | {check}
 | {check}
 
-| {sles}-ES 8
-| {x86_64}
+| {sles} for SAP 15
+| {x86_64}, {ppc64le}
 | {check}
 | {check}
 
-| {sles}-ES 7
-| {x86_64}
+| {sles} for SAP 12
+| {x86_64}, {ppc64le}
 | {check}
 | {check}
 
-| {sles} for SAP 15
-| {x86_64}, {ppc64le}
-| {check}
+| {micro}
+| {x86_64}, {ppc64le}, {aarch64}
+| {cross}
 | {check}
 
-| {sles} for SAP 12
-| {x86_64}, {ppc64le}
+| {opensuse} Leap 15
+| {x86_64}, {aarch64}
 | {check}
 | {check}
 
-| {rhel} 8
+| {sles} ES 8
 | {x86_64}
 | {cross}
 | {check}
 
-| {rhel} 7
+| {sles} ES 7
 | {x86_64}
 | {check}
 | {check}
 
-| CentOS 8
-| {x86_64}
+ifeval::[{uyuni-content} == true]
+| {alibabaclo} 2
+| {x86_64}, {aarch64}
 | {cross}
 | {check}
+endif::[]
 
-| CentOS 7
+| {almalinux} 8
 | {x86_64}
```

Comment:
You are missing aarch64 here. The original issue says "Document aarch64 for CentOS7/8, Oracle7/8, Rocky Linux 8, AlmaLinux 8, Amazon Linux2, openSUSE Leap 15.3". Review the others OS, in case I am missing some more.

Reply:
Added after review.